src/design-system/settings/setting.css.ts

```diff
@@ -0,0 +1,27 @@
+import { style, vars } from '../../design-tokens';
+
+export const root = style({});
+
+export const separator = style({
+  width: '100%',
+  height: 1,
+  background: vars.colors.$settings_item_separator_color,
+  selectors: {
+    [`${root}:first-of-type &`]: {
+      background: 'none',
+    },
+    [`${root}:hover &`]: {
+      background: 'none',
+    },
+  },
+});
+
```

Comment:
Suggestions: Maybe you can use `Divider` https://github.com/input-output-hk/lace-ui-toolkit/tree/main/src/design-system/divider

Reply:
yeah good point, i thought that it might be more work to make it react to these selectors then?